### PR TITLE
Add Toric Code State Prep to ReCirq (#3)

### PR DESCRIPTION
**Pull request by dstrain115**
PR #3 of Toric code experiment.

### CODE REVIEW

File: recirq/toric_code/toric_code_state_prep.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+from __future__ import annotations
+
+from typing import Iterator, List, Set
+
+import cirq
+
+from . import toric_code_rectangle as tcr
+
+
+class ToricCodeStatePrep:
```
Comment (mpharrigan) [inline recirq/toric_code/toric_code_state_prep.py:23]: does this need to be a class? just use free functions

Comment (dstrain115) [reply recirq/toric_code/toric_code_state_prep.py]: Changed to free functions.

File: recirq/toric_code/toric_code_state_prep.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+from __future__ import annotations
+
+from typing import Iterator, List, Set
+
+import cirq
+
+from . import toric_code_rectangle as tcr
+
+
+def toric_code_cnot_circuit(code:tcr.ToricCodeRectangle, x_basis: bool = False) -> cirq.Circuit:
```
Comment (mpharrigan) [inline recirq/toric_code/toric_code_state_prep.py:28]: can you run this file through the formatter of your choice to get the spacing correct

**Review by mpharrigan — APPROVED** [review]
lgtm, but please pep8 format

**Pull request merged**